frontend/src/actionCreators/authActionCreator.tsx

```diff
@@ -0,0 +1,25 @@
+import { showLoading, hideLoading } from "react-redux-loading-bar";
+import { request, success, error } from "actions/genericActions";
+import * as reducerTypes from "constants/reducerTypes";
+import * as API from "constants/API";
+import { ENVIRONMENT } from "constants/environment";
+import CustomAxios from "customAxios";
+import { AxiosResponse } from "axios";
+import { createRequestHeader } from "utils/RequestHeaders";
+
+export const getActivateUserAction = () => (dispatch: Function) => {
```

Comment:
Might only want to send this request if the user doesn't have any roles.

Reply:
Will leave this for now do to the negligible performance impact.